lib/state_manager/adapters/active_record.rb

```diff
@@ -67,11 +67,11 @@ def self.included(base)
 
             # In the AR use case, we don't want to run any callbacks
             # until the model has been saved
-            def run_before_callbacks(*args)
+            def run_before_callbacks(*args, **kwargs)
               self.pending_transition = args
```

Comment:
seeing lines like
```
          _run_before_callbacks(*pending_transition)
```
makes me think maybe we need to do soemthing more like:
```
            def run_before_callbacks(*args, **kwargs)
              self.pending_transition_args = args
              self.pending_transition_kwargs = kwargs
```
and make the appropriate changes elsewhere

Reply:
Good point! I think this will actually all be positional arguments so probably ok to forgo a **kwargs here altogether.